dev-tools/packer/platforms/windows/uninstall-service.ps1.j2

```diff
@@ -1,5 +1,7 @@
 # delete service if it exists
 if (Get-Service {{.beat_name}} -ErrorAction SilentlyContinue) {
   $service = Get-WmiObject -Class Win32_Service -Filter "name='{{.beat_name}}'"
+  $service.StopService()
+  Start-Sleep -s 1
```

Comment:
I assume StopService happens asynchronously and that's why we need the sleep, right? One second might not be enough for the Beat to stop, so ideally we wouldn't rely on a sleep here, but if there's no way of waiting for StopService to be done, I'm fine with this solution and the choses 1s interval.

Reply:
Will test it.
The StopService call itself should wait for the executable to stop.
The 1 second is for - on windows - it can take slightly longer for the executable to actually terminate.

Note - the call in install-service scripts is exactly the same - if there is an issue here 0 it is more likely to be uncovered there than as part of the uninstall script ...

Reply:
Question: Is there somewhere that I can get the built artifacts (ie. the beats packages) from? Or do I need to build it all locally?